### ISSUE TITLE
Error while installing requirement.txt on colab

### ISSUE THREAD

**Issue by aakashag1** [saifhassan/Wav2Lip-HD#15]
Getting the below error when running the code in colab with python version Python 3.10.12 and pip version pip 23.1.2: 

Building wheels for collected packages: librosa, numba, llvmlite
  Building wheel for librosa (setup.py) ... done
  Created wheel for librosa: filename=librosa-0.7.0-py3-none-any.whl size=1598349 sha256=92423d72b10caddb793db855ad54822fa57e1d4a4173993974b59fc1fa84487d
  Stored in directory: /root/.cache/pip/wheels/6e/4d/e1/f2c74cab7e013fb674ac6102148d0620c0ece2d9d24066a594
  error: subprocess-exited-with-error
  
  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  Building wheel for numba (setup.py) ... error
  ERROR: Failed building wheel for numba
  Running setup.py clean for numba
  error: subprocess-exited-with-error
  
  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  Building wheel for llvmlite (setup.py) ... error
  ERROR: Failed building wheel for llvmlite
  Running setup.py clean for llvmlite
Successfully built librosa
Failed to build numba llvmlite
ERROR: Could not build wheels for numba, llvmlite, which is required to install pyproject.toml-based projects

Please help how to solve this issue!

**Comment by Forrestbeta** [comment]
same error

**Comment by peja22** [comment]
any chance to run it on colab?

**Comment by Nifty0x** [comment]
same error here. anyone knows how to solve it?

**Comment by saifhassan** [comment]
> any chance to run it on colab?

Unfortunately, we have not provided any notebook file for colab.

**Comment by saifhassan** [comment]
> Getting the below error when running the code in colab with python version Python 3.10.12 and pip version pip 23.1.2:
> 
> Building wheels for collected packages: librosa, numba, llvmlite Building wheel for librosa (setup.py) ... done Created wheel for librosa: filename=librosa-0.7.0-py3-none-any.whl size=1598349 sha256=92423d72b10caddb793db855ad54822fa57e1d4a4173993974b59fc1fa84487d Stored in directory: /root/.cache/pip/wheels/6e/4d/e1/f2c74cab7e013fb674ac6102148d0620c0ece2d9d24066a594 error: subprocess-exited-with-error
> 
> × python setup.py bdist_wheel did not run successfully. │ exit code: 1 ╰─> See above for output.
> 
> note: This error originates from a subprocess, and is likely not a problem with pip. Building wheel for numba (setup.py) ... error ERROR: Failed building wheel for numba Running setup.py clean for numba error: subprocess-exited-with-error
> 
> × python setup.py bdist_wheel did not run successfully. │ exit code: 1 ╰─> See above for output.
> 
> note: This error originates from a subprocess, and is likely not a problem with pip. Building wheel for llvmlite (setup.py) ... error ERROR: Failed building wheel for llvmlite Running setup.py clean for llvmlite Successfully built librosa Failed to build numba llvmlite ERROR: Could not build wheels for numba, llvmlite, which is required to install pyproject.toml-based projects
> 
> Please help how to solve this issue!

Error in installations, I would suggest to uninstall/remove that environment completely and install again.

**Comment by fxs-space** [comment]
> > Getting the below error when running the code in colab with python version Python 3.10.12 and pip version pip 23.1.2:
> > Building wheels for collected packages: librosa, numba, llvmlite Building wheel for librosa (setup.py) ... done Created wheel for librosa: filename=librosa-0.7.0-py3-none-any.whl size=1598349 sha256=92423d72b10caddb793db855ad54822fa57e1d4a4173993974b59fc1fa84487d Stored in directory: /root/.cache/pip/wheels/6e/4d/e1/f2c74cab7e013fb674ac6102148d0620c0ece2d9d24066a594 error: subprocess-exited-with-error
> > × python setup.py bdist_wheel did not run successfully. │ exit code: 1 ╰─> See above for output.
> > note: This error originates from a subprocess, and is likely not a problem with pip. Building wheel for numba (setup.py) ... error ERROR: Failed building wheel for numba Running setup.py clean for numba error: subprocess-exited-with-error
> > × python setup.py bdist_wheel did not run successfully. │ exit code: 1 ╰─> See above for output.
> > note: This error originates from a subprocess, and is likely not a problem with pip. Building wheel for llvmlite (setup.py) ... error ERROR: Failed building wheel for llvmlite Running setup.py clean for llvmlite Successfully built librosa Failed to build numba llvmlite ERROR: Could not build wheels for numba, llvmlite, which is required to install pyproject.toml-based projects
> > Please help how to solve this issue!
> 
> Error in installations, I would suggest to uninstall/remove that environment completely and install again.

Reinstalled, same error reported

**Comment by Nifty0x** [comment]
Yeah, same error when reinstalled. What's your suggestion @saifhassan?

**Comment by saifhassan** [comment]
Try Colab Version by Wav2Lip Official Repo:

https://colab.research.google.com/github/anothermartz/Easy-Wav2Lip/blob/Fast/EZWav2Lip_v5.ipynb#scrollTo=KS7vOhVWhgR7

**Comment by AIhasArrived** [comment]
Hello @saifhassan , this one is no longer available, what do you recommand using nowadays?
Please answer.

**Comment by start-life** [comment]
This is my problem too

**Comment by Heaverno** [comment]
numba 0.48 only supports python 3.6/3.7/3.8
maybe you need to specify a python version somewhere @saifhassan

**Comment by Mukrosz** [comment]
Heya, running into the same issue with python3.11. Is the python version not supported?
